sw/device/lib/dif/dif_otbn.h

```diff
@@ -255,6 +256,17 @@ dif_otbn_result_t dif_otbn_is_busy(const dif_otbn_t *otbn, bool *busy);
 dif_otbn_result_t dif_otbn_get_err_bits(const dif_otbn_t *otbn,
                                         dif_otbn_err_bits_t *err_bits);
 
+/**
+ * Get the instruction counter value.
```

Comment:
Let's make this more helpful to a user who doesn't exactly how what the instruction counter is:

```suggestion
 * Get the number of instructions executed in the most recent OTBN run.
```

Reply:
Since it's possible to read from this in the middle of the run, maybe "Get the number of instructions executed so far in the current OTBN run if there is one, or the number executed in total in the previous OTBN run." ?